packages/fiori/src/MediaGalleryTemplate.tsx

```diff
@@ -0,0 +1,50 @@
+import type MediaGallery from "./MediaGallery.js";
+import MediaGalleryItem from "./MediaGalleryItem.js";
+import Carousel from "@ui5/webcomponents/dist/Carousel.js";
+import Button from "@ui5/webcomponents/dist/Button.js";
+
+export default function MediaGalleryTemplate(this: MediaGallery) {
+	return (
+		<div class="ui5-media-gallery-root">
+			<div class="ui5-media-gallery-display-wrapper">
+				<div class="ui5-media-gallery-display"
+					onClick={this._onDisplayAreaClick}>
+					{this._isPhonePlatform
+						? <Carousel
+							onNavigate={this._onCarouselNavigate}
+							hideNavigationArrows
```

Comment:
really minor - explicitly setting to true is recommended => `hideNavigationArrows={true}`